### PR TITLE
PLAT-132845: Update focus rule when focusableScrollber is byEnter

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
There are 2 changes in the new UX for the 2022TV when `focusableScrollbar is byEnter`
1. If `'byEnter'`, scroll thumb will get focus first by directional keys, then the scroll body will get focus by enter key or back key pressed on scroll thumb.
2. If the thumb is at the end of the scroll, spotlight can escape with the arrow keys.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Partially revert old PR #728 #744 
- Set spotlight filter on scroller body firstly instead of scroll thumb
- Make it possible to escape from thumb after checking i the thumb reached the end of the scroll

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-132845

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 - `sandstone/Item` public class names `itemContent`, `content`, and `label`
 - `sandstone/Scroller` prop `scrollbarTrackCss` to customize scroll track and thumb style
 
+### Changed
+
+- `sandstone/Scroller` focus rule to match latest UX when `focusableScrollbar` prop is `byEnter`
```
Comment (MikyungKim) [inline CHANGELOG.md:23]: Please put the change log in the right place.

Comment (jeonghee27) [reply CHANGELOG.md]: Sorry. Git drove me into a corner.

File: Scroller/useThemeScroller.js [hunk 2]
```diff
@@ -103,6 +86,9 @@ const getFocusableBodyProps = (scrollContainerRef, contentId, isScrollbarVisible
 		}
 	};
 
+	// Initial filter setting
+	setNavigableFilter({filterTarget: 'body'});
```
Comment (MikyungKim) [inline Scroller/useThemeScroller.js:90]: I'm afraid this be calling whenever Scroller got re-rendered. Shouldn't we put this in `useEffect` with [] dependency?

**Review by MikyungKim — APPROVED** [review]
LGTM!

**Pull request merged**